### DISCUSSION TITLE
Billing details for flutter web payment element.

### DISCUSSION THREAD

**Discussion by ankitmiyantechspian** [flutter-stripe/flutter_stripe#1464] [Q&A]
Hello @remonh87, I am trying to make a USD payment from India and as paying USD from India we need to give the customer billing address in the mobile application flow i am able to do the same by passing BillingDetails, BillingDetailsCollectionConfiguration and IntentConfiguration but could you tell me how to achieve the same for web?
The attached SS is for mobile configuration

<img width="544" alt="Screenshot 2023-10-26 at 1 52 52 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/137860496/80d49d79-8d7f-4566-adb6-37e283b460c2">

**Comment by remonh87** [comment]
Hi you can add the billingdetails to the the paymentintent and it will added to the payment.

**Reply by ankitmiyantechspian** [reply]
Hello, @remonh87 but how do I do this for the web?
In the previously given code snipped I am adding this configuration and my payment sheet for the mobile app is coming up with card details form and billing address form eg address, state, city, country, and pin. Still, for the Web, I can't see a way to enable the billing address form with PaymentElement. The Payment Element is only showing me the card form and country dropdown i want to show the billing address form along with it in the Payment Element web like the payment sheet on mobile please help.
Please see the attachments for your reference.
<img width="1063" alt="Screenshot 2023-10-31 at 2 12 58 PM" src="https://github.com/flutter-stripe/flutter_stripe/assets/137860496/116c0df4-524b-4676-a743-f4baa66b66f0">
![simulator_screenshot_252BB7B9-6D8F-4E37-9BF0-B6FC1E00AC07](https://github.com/flutter-stripe/flutter_stripe/assets/137860496/e8c5a37b-67f0-490b-be28-1cbffe102336)
